sale_order_type/tests/test_sale_order_type.py

```diff
@@ -19,9 +19,22 @@ def setUp(self):
         )
         self.account_model = self.env["account.account"]
         self.user_type_id = self.env.ref("account.data_account_type_revenue")
-        self.account = self.account_model.create(
-            {"code": "410000", "name": "Income", "user_type_id": self.user_type_id.id}
+        self.account = self.account_model.search(
```

Comment:
This is happening because tests are not post-install. Change them, and don't create ever an account. Just look for an account of the indicated type (no restricting to the code).